src/setupProxy.js

```diff
@@ -18,8 +17,37 @@ const {
   REACT_APP_SEARCH_URL,
   REACT_APP_HIGHLIGHTS_URL,
   REACT_APP_OS_WEB_API_URL,
+  REACT_APP_ARCHIVE_URL,
 } = require('./config');
 
+// lots of stuff relies on this
+const JSDOM = require('jsdom').JSDOM;
```

Comment:
all this seems duplicated from entry.js, this is the third babel config in the project. we should find a way to share this with the entry.js config